test/common.bash

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+
+# bats setup function
+setup() {
+  export KUBECONFIG=$(mktemp)
+  export XDG_CACHE_HOME="$(mktemp -d)"
+}
+
+# bats teardown function
+teardown() {
+  rm -f $KUBECONFIG
+  rm -f $XDG_CACHE_HOME/kubectx
+  rmdir $XDG_CACHE_HOME
```

Comment:
you can combine all these into one `rm -rf $XDG_CACHE_HOME`, if you put kubeconfig in that dir, too.